### DISCUSSION TITLE
Exercise

### DISCUSSION THREAD

**Discussion by Morehman27** [skills/.github#630] [Test with Actions]
What is GitHub Copilot?

 

GitHub Copilot is an AI pair programmer that you can use to get code suggestions.

GitHub Copilot is OpenAI Codex, a new AI system created by OpenAI.

GitHub Copilot is a JavaScript public repository and is one of the best supported languages.

GitHub Copilot can write a comment describing logic and you can add your suggested code to implement the solution.

2. 

What are the supported integrated development environment extensions for GitHub Copilot?

 

Visual Studio Code and Visual Studio

GitHub.com, Visual Studio Code, Visual Studio, Neovim, and JetBrains

Visual Studio Code, Visual Studio, Neovim, and JetBrains

3. 

What is the difference between GitHub Copilot Business and GitHub Copilot Enterprise?

 

GitHub Copilot Enterprise has code completions, while GitHub Copilot Business does not.

GitHub Copilot Enterprise has chat in IDE and mobile, while GitHub Copilot Business does not.

GitHub Copilot Enterprise has an extra layer of personalization, with organization utilizing their own codebase to train GitHub Copilot.

GitHub Copilot Enterprise has an extra layer of security, with IP indemnity and enterprise-grade security, safety, and privacy.

**Comment by vnjvijay** [comment]
1. GitHub Copilot is an AI pair programmer that you can use to get code suggestions.
2. Visual Studio Code, Visual Studio, Neovim, and JetBrains
3. GitHub Copilot Enterprise has an extra layer of personalization, with organization utilizing their own codebase to train GitHub Copilot.
